frontend-ssr/services/ExerciseSubmissions.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Submission } from '~/redux/exercise-submissions/types';
+import HttpService from './Http.service';
+
+// FIXME ---> we need to do a little more research to find a better solution
+
+class ExerciseService {
+  static getSubmissions(page = 0, query = '') : Promise<Submission[]> {
+    return HttpService.get(`${process.env.ENDPOINT}/exercises/submissions?search=al${page}&${query}`) as unknown as Promise<Submission[]>;
```

Comment:
Probably in a hurry but you left `?search-al` in the request string. I put it there just as an example. 

So it should be `/submissions?page=${page}&query=${query}` - this is how query params work